docs/source/contributor-guide/benchmarking.md

```diff
@@ -0,0 +1,43 @@
+# Comet Benchmarking Guide
```

Comment:
license header?

Reply:
Good catch. Looks like we have no CI check for that for docs changes. I will look into that